.gitignore

```diff
@@ -15,7 +15,6 @@ bundle.map
 **/Pods/
 **/xcuserdata/
 build/
-Podfile.lock
```

Comment:
Unsure why this was being ignored? It should normally be checked in to ensure consistent builds.

Reply:
Can you remove this. I have mentioned the reason in the comment on the Podfile.lock

Reply:
Done 👍